src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -579,11 +579,7 @@ public async Task Initialize(HostWaitLoop waitLoop, CancellationToken token)
 
             try
             {
-                Results res = await this.MICommandFactory.ListTargetFeatures();
-                if (res.Contains("async"))
-                {
-                    await this.MICommandFactory.EnableTargetAsyncOption();
-                }
+                await this.MICommandFactory.EnableTargetAsyncOption();
```

Comment:
Add a note why we can't do the check

Reply:
This check failed because it was done too early.
Needs to come after the initialize commands.